code/datums/traits/negative.dm

```diff
@@ -509,6 +510,7 @@
 		reagent_type = prot_holder.type
 	reagent_instance = new reagent_type()
 	H.reagents.addiction_list.Add(reagent_instance)
+	junkie_warning = ("You thought you kicked it, but you suddenly feel like you need [reagent_instance.name] again...")
```

Comment:
this will overwrite the drunkard one, only do this if it sees that junkie_warning isn't set

Reply:
I didn't realize it'd work that way, thanks for letting me know!